### DISCUSSION TITLE
part-ii/2021-97/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#371] [Question comments]
Part II 2021 Mathematics of Machine Learning: Paper 1, Section II, J 

https://questions.tripos.org/part-ii/2021-97/

**Comment by ArthurConmy** [comment]
<details>
<summary>b)</summary>

Functions in $ \mathcal{F}' $ look like 

$$ \underbrace{x_1^2 + ... + x_d^2}_{\text{ One basis element }} - \underbrace{2c_1x_1 - ... - 2c_d x_d}_{\text{ Spanned by $d$ functions $ x \mapsto x_i $ }} + \tilde{c} $$

where $\tilde{c}$ is a constant so spanned by one further basis element, so we indeed get dimension $d+2$.

</details>
